packages/components/Badge/src/Badge.types.ts

```diff
@@ -49,6 +57,11 @@ export interface BadgeConfigurableProps {
    * @defaultvalue absolute
    */
   position?: FlexStyle['position'];
+
+  /**
+   * Sets shadow style with `ambient` and `key` props
+   */
+  shadowToken?: ShadowToken;
```

Comment:
For clarification, we did decide that shadow is a prop and a token? Or just a token? I don't think it makes sense for a prop to have the name "token" in it, because that signifies that it's not supposed to be a prop.

Reply:
I still don't understand why shadow is a prop and not just a token, could you explain why?